job/tests/test_views.py

```diff
@@ -0,0 +1,230 @@
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.test import APITestCase
+from job.models import Job
+from utilities.utilities import slug_helper
+
+User = get_user_model()
+
+
+class JobTests(APITestCase):
+    def setUp(self):
+        url = "/authnz/login_email/"
+        # Creating test superuser
+        user_data = {
+            "username": "testsuperuser",
+            "email": "testsuperuser@gmail.com",
+            "password": "testuser123",
+        }
+        self.user = User.objects.create_superuser(**user_data)
+        self.user.profile.email_confirmed = True
+        self.user.profile.email = "testsuperuser@gmail.com"
+        self.user.profile.save()
+
+        # Getting token
+        self.token = self.client.post(url, user_data, format="json").json()["data"]["token"]
+
+        # Creating job
+        Job.objects.create(
+            name="برنامه نویس وب",
+            job_slug=slug_helper("برنامه نویس وب"),
+            cover="/default/system/cover.jpg",
+            icon="fas fa-home",
+            description="توسعه دهنده و برنامه نویس وب",
+            is_deleted=False,
+            approved=False,
+        )
+
+        Job.objects.create(
+            name="برنامه نویس بک اند",
+            job_slug=slug_helper("برنامه نویس بک اند"),
+            cover="/default/system/cover.jpg",
+            icon="fas fa-home",
+            description="توسعه دهنده و برنامه نویس بک اند",
+            is_deleted=False,
+            approved=False,
+        )
+
+    def test_create_job(self):
+        url = "/job/create/"
+        data = {
+            "name": "برنامه نویس پایتون",
+            "cover": "/default/system/cover.jpg",
+            "icon": "fas fa-home",
+            "description": "برنامه نویس پایتون",
+        }
+        self.client.credentials(HTTP_AUTHORIZATION="JWT " + self.token)
+        response = self.client.post(url, data, format="json")
+        job = Job.objects.last()
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response._headers["content-type"][1], "application/json")
+        self.assertEqual(Job.objects.count(), 3)
+        self.assertEqual(job.name, "برنامه نویس پایتون")
+        self.assertEqual(job.job_slug, slug_helper("برنامه نویس پایتون"))
+
+    def test_job_listview(self):
```

Comment:
Suggestion: to have integrity in names try use a format like test_instance_functionality_view_other_description: test_job_list_view